CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
+## Version 6.0.0
+
+### Changed
+
+* [214](https://github.com/appfolio/ae_page_objects/pull/214) Add Ruby 3 Support
+
+  This removes support for Ruby versions below `2.7.7` and adds in `3.0.X`, `3.1.X`, and `3.2.X`. There are no external API changes, just internal updates to ensure compatibility with changes to Ruby 3 splat operators and keyword argument behaviors.
```

Comment:
I think the changelog entry here can be the same as the PR comment

Reply:
Fixed